lib/ini.js

```diff
@@ -4,27 +4,46 @@ const encode = (obj, opt = {}) => {
   if (typeof opt === 'string') {
     opt = { section: opt }
   }
-  opt.whitespace = opt.whitespace === true
+  opt.align = opt.align === true
   opt.newline = opt.newline === true
+  opt.sort = opt.sort === true
+  opt.whitespace = opt.whitespace === true || opt.align === true
   /* istanbul ignore next */
   opt.platform = opt.platform || process?.platform
 
   /* istanbul ignore next */
   const eol = opt.platform === 'win32' ? '\r\n' : '\n'
   const separator = opt.whitespace ? ' = ' : '='
   const children = []
+
+  const sort = opt.sort
+  const keys = sort ? Object.keys(obj).sort() : Object.keys(obj)
+
+  // padToChars is determined as follows:
+  // 1. Get the keys
+  // 2. Exclude keys pointing to objects unless the value is null or an array
+  // 3. Add `[]` to array keys
+  // 4. Ensure non empty set of keys
+  // 5. Reduce the set to the longest key
+  // 6. Get the length
+  const padToChars = opt.align
```

Comment:
Nested ternaries are very hard to follow and prone to errors.  Can this be a `let` that defaults to `0` and an if statement that changes it if `opt.align` is set?

Reply:
Yep. I was being a little lazy here I guess. JS is not my first or second language so always happy to accept coding style recommendations.